buildpacks/builder.go

```diff
@@ -21,6 +20,12 @@ import (
 	"github.com/buildpacks/pack/pkg/logging"
 )
 
+// DefaultBuilderImages for Pack builders indexed by Runtime Language
+var DefaultBuilderImages = map[string]string{
+	"node": "gcr.io/paketo-buildpacks/builder:base",
+	"go":   "gcr.io/paketo-buildpacks/builder:base",
+}
```

Comment:
This is a lazily-poplated list of defaults as they are given s2i support (and thus can not rely on `manifest.yaml` only). 
This list will eventually expand to include all Paketo builders for runtimes we include.
For language packs which provide entirely new runtimes, the default will need to be provided in a `manifest.yaml` bu the language pack developer.